lemmy_apub/src/inbox/user_inbox.rs

```diff
@@ -50,31 +51,44 @@ pub async fn user_inbox(
 ) -> Result<HttpResponse, LemmyError> {
   let activity = input.into_inner();
   let username = path.into_inner();
+  let user = blocking(&context.pool(), move |conn| {
+    User_::read_from_name(&conn, &username)
+  })
+  .await??;
+
+  let to = activity
+    .to()
+    .context(location_info!())?
+    .to_owned()
+    .single_xsd_any_uri();
+  if Some(user.actor_id()?) != to {
+    return Err(anyhow!("Activity delivered to wrong user").into());
+  }
```

Comment:
Isn't one of these `actor_id/inbox`, and the other is just `actor_id`?

Reply:
No I changed all the `to` to have the actor_id, just like the examples in the spec. I dont think it makes sense to have the inbox as destination, after all it belongs to the actor.

https://www.w3.org/TR/activitypub/#create-activity-outbox